src/js/lib/admin-page/components/SiteDetails.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+const SiteDetails = ( { apikey, postsToTrack, pagesToTrack, phpVersion, pluginVersion } ) => {
+	const [ isCopied, setIsCopied ] = useState( false );
+	const copyToClipboard = () => {
+		const el = document.createElement( 'textarea' );
+		el.value = `Site ID: ${ apikey }, PHP Version: ${ phpVersion }, Post Types to track: ${ postsToTrack }, Pages to Track: ${ pagesToTrack }, Plugin Version: ${ pluginVersion }`;
+		el.setAttribute( 'readonly', '' );
+		el.style.position = 'absolute';
+		el.style.left = '-9999px';
+		document.body.appendChild( el );
+		el.select();
+		document.execCommand( 'copy' );
+		document.body.removeChild( el );
+		setIsCopied( true );
+	};
+	return (
+		<div className="site-details-container">
+			<div className="site-detail">
+				<div className="details-label">
+					<span>{ __( 'Site Details', 'wp-parsely' ) }</span>
+				</div>
+				<div className="details-info">
+					<span className="details-info-item">
+						{ __( 'Parsely Site ID: ', 'wp-parsely' ) } { apikey }
```

Comment:
Don't include the trailing space in the translatable string, here, or any of the instances below.